### PR TITLE
Specify pulp2_distributor_repository_ids instead of distributor_ids

### PR DESCRIPTION
**Pull request by dralley**
closes: #5837
https://pulp.plan.io/issues/5837

### CODE REVIEW

**Comment by dralley** [comment]
Not done yet, just trying to get the general approach worked out.  I'm attempting to make it work without pre-processing the distributor repos.

File: pulp_2to3_migration/app/models/base.py [hunk 1]
```diff
@@ -203,10 +203,15 @@ def _parse_plugin_plan(self, repository_data):
...
 
-                    distributor_ids = repository_version.get('distributor_ids', [])
-                    self.repositories_distributors_to_migrate.extend(distributor_ids)
+                    distributor_repo_ids = repository_version.get(
+                        'pulp2_distributor_repository_ids', []
+                    )
+                    self.repositories_distributors_to_migrate.extend(distributor_repo_ids)
+
+                    repository_versions.append(
+                        {'repo_id': pulp2_repository_id, 'dist_repo_ids': distributor_repo_ids}
```
Comment (dralley) [inline pulp_2to3_migration/app/models/base.py]: Only using a dict temporarily

**Comment by jlsherrill** [comment]
how does this work if a repository has multiple distributors?

**Comment by ipanova** [comment]
@jlsherrill there will be created 2+ distributions ( assuming those are 2+ distributors we migrate unlike the rsync_distributor, for example)

**Comment by ipanova** [comment]
> how does this work if a repository has multiple distributors?
> 
> EDIT: for example, a yum repo would have a yum_distributor, an export_distributor, and a yum_clone_distributor (custom katello one), we only care about migrating the first.

Only distributors that are specified in the migrator will be recognized and migrated example: https://github.com/pulp/pulp-2to3-migration/blob/master/pulp_2to3_migration/app/plugin/iso/migrator.py#L41

**Comment by jlsherrill** [comment]
@ipanova thanks!  all looks good, as long as anything not recognized is ignored

File: pulp_2to3_migration/app/plugin/iso/repository.py [hunk 2]
```diff
@@ -69,6 +69,8 @@ class IsoDistributor(Pulp2to3Distributor):
         base_config = cls.parse_base_config(pulp2distributor, pulp2_config)
         base_config['base_path'] = pulp2_config.get(
             'relative_url', pulp2distributor.pulp2_repository.pulp2_repo_id)
+        # This fails because of course there is no pulp2_repository set on lone distributors.
+        # So what should be used instead?
```
Comment (dralley) [inline pulp_2to3_migration/app/plugin/iso/repository.py]: @goosemania This PR should be almost done, but I'm not sure how we want to set up the ```base_path``` in this instance.

Comment (goosemania) [reply pulp_2to3_migration/app/plugin/iso/repository.py]: @dralley ,
s/`pulp2distributor.pulp2_repository.pulp2_repo_id`/`pulp2distributor.pulp2_repo_id`/

There is pulp2_repo_id stored on the distributor model. It was added with the recent PR, same issue was for the importers.

Comment (dralley) [reply pulp_2to3_migration/app/plugin/iso/repository.py]: Cool, that was the last issue. It seems to work now

**Pull request merged**